L03-generics/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,31 @@
+package homework;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+public class CustomerService {
+    SortedMap<Customer, String> customersTreeMap =
```

Comment:
1) private final
2) тип лучше объявлять как интерфейс
3) посмотрите на `NavigableMap`

```suggestion
    private final NavigableMap<Customer, String> customersTreeMap =
```